Fixes unknown method .to_json_pointer

philsturgeon opened the pull request:
Hello! Brilliant library, thanks for making it. 

I was using your main branch to avoid depdendecy conflict between this repo and openapi_first, and I was getting an error about this string (which is already a JSON Pointer) so figured I would try removing the method and it's worked.

mkon commented:
Hi @philsturgeon, can you supply an example OAS definition where this issue appears? I suspect there is a issue somewhere else in the included parser, since `tgt` at this point should be a `OpenapiContracts::Doc::Pointer` which does provide the method you are removing.

philsturgeon commented:
@mkon convenitently I'm building some sample code so I've got a full blown reproduction repo over here: https://github.com/philsturgeon/rails-design-first 

If you swap out my fork in the Gemfile for your main branch you should see the issue if you try running rspec.

mkon commented:
Ok the issue is that the parser currently does not take into account that there could be multiple paths to the same file, so the pointer could not be found (eg `./foo.yaml` vs `foo.yaml`). I am closing, the correct fix is #84

philsturgeon commented:
Thank you for figuring it out and fixing it with a far better fix than my cheeky hack. I've deleted my fork and switched the sample base over to use your repo and the main branch. If you could tag a version shortly I'll update the same code and get our guide published. 

Thanks for making this excellent tool!

mkon commented:
You're welcome, I released version `0.13.0`

philsturgeon commented:
Thanks! The guide is over here https://github.com/bump-sh/docs/pull/162/ and will show up on https://docs.bump.sh/guides/openapi/ when it's done, if that's of any interest to you. 🙌🏻
